crates/shared/src/price_estimation/sanitized.rs

```diff
@@ -0,0 +1,245 @@
+use super::{ensure_token_supported, Estimate, PriceEstimating, PriceEstimationError, Query};
+use crate::bad_token::BadTokenDetecting;
+use anyhow::Result;
+use futures::future;
+use model::order::BUY_ETH_ADDRESS;
+use primitive_types::{H160, U256};
+use std::sync::Arc;
+
+const ETH_UNWRAP_COST: U256 = U256([0, 0, 0, 123]);
```

Comment:
The total gas cost (minus the tx overhead) is 
```
35192 - 21000 = 14192
```

This is based on the following transaction: https://etherscan.io/tx/0x50e67d83c14d73ec67a3abdb720350cfe7facd232d3277739be87304e0a82314

We could be more accurate and get average out the total gas used on every unwrap, but I think that these kinds of transactions are nearly the same.

Let me know if we want to use a value based on a larger sample size.